doc/source/reference/c-api.array.rst

```diff
@@ -382,10 +382,11 @@ From other objects
     sequence, or object that exposes the array interface, *op*. The
     parameters allow specification of the required *dtype*, the
     minimum (*min_depth*) and maximum (*max_depth*) number of
-    dimensions acceptable, and other *requirements* for the array. The
-    *dtype* argument needs to be a :c:type:`PyArray_Descr` structure
+    dimensions acceptable, and other *requirements* for the array. This
+    function **steals a reference** to the dtype argument, which needs
```

Comment:
Is the strong emphasis needed. Seem more like a personal reaction here :)

Reply:
Maybe :)
However, must of the "steals a reference" in the doc uses bold.